### DISCUSSION TITLE
EE6 - Nested categories have cascading CSS issues?

### DISCUSSION THREAD

**Discussion by JasonBoothman** [ExpressionEngine/ExpressionEngine#2914] [Feature Requests]
This may be intended, but the styling on nested categories looks odd with the indentation as well as the borders. You can view this anywhere categories are displayed and include multiple levels (Parent Category field when editing a category, Editing an entry with categories assigned to it, etc.).

The nested categories are indented, have a border on the left, and in some cases (if you go three levels deep) have a bottom border as well, which servers to drop odd shapes in the field which is very distracting (see attached photo).

Update: This seems to be any field that has a nested checkbox or radio button display.

![category-management-borders](https://user-images.githubusercontent.com/4921348/96471079-f08aef80-11fc-11eb-8e69-a36913a76fac.png)

**Comment by ops-andy** [comment]
confirmed that I see the same styling

**Comment by intoeetive** [comment]
@Yulyaswan can you check this and either fix or close this issue?

**Comment by Yulyaswan** [comment]
For me, it looks like on screenshot.
It would be nice to hear @ajohnson opinion on styling this part.
<img width="634" alt="Screenshot 2022-11-30 at 12 50 14" src="https://user-images.githubusercontent.com/23382425/204800848-4f18008e-f661-4a47-adc7-728b2d2c1193.png">

**Comment by Yulyaswan** [comment]
I can't reproduce those styles
everywhere have like on prev screenshot
Closing this issue
